content-src/asrouter/templates/StartupOverlay/StartupOverlay.jsx

```diff
@@ -2,107 +2,49 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { actionCreators as ac, actionTypes as at } from "common/Actions.jsm";
-import { connect } from "react-redux";
+import { actionCreators as ac } from "common/Actions.jsm";
 import React from "react";
 
-const FLUENT_FILES = [
-  "branding/brand.ftl",
-  "browser/branding/sync-brand.ftl",
-  "browser/newtab/onboarding.ftl",
-];
-
-export class _StartupOverlay extends React.PureComponent {
+export class StartupOverlay extends React.PureComponent {
   constructor(props) {
     super(props);
     this.onInputChange = this.onInputChange.bind(this);
     this.onSubmit = this.onSubmit.bind(this);
     this.clickSkip = this.clickSkip.bind(this);
-    this.initScene = this.initScene.bind(this);
     this.removeOverlay = this.removeOverlay.bind(this);
     this.onInputInvalid = this.onInputInvalid.bind(this);
 
     this.utmParams =
       "utm_source=activity-stream&utm_campaign=firstrun&utm_medium=referral&utm_term=trailhead-control";
 
     this.state = {
+      show: false,
       emailInput: "",
-      overlayRemoved: false,
-      deviceId: "",
-      flowId: "",
-      flowBeginTime: 0,
     };
     this.didFetch = false;
```

Comment:
`this.didFetch` is still here and looks like there's a `this.state.flowId` incorrectly being used and failing now that we removed the state.